llmcord/llm_tools.py

```
@@ -0,0 +1,190 @@
+                    "Accept-Language": "en-US,en;q=0.9",
+                    "Accept-Encoding": "gzip, deflate, br",
+                    "Referer": "https://www.google.com/",
+                    "Connection": "keep-alive",
+                },
+            )
+            r.raise_for_status()
+            content = trafilatura.extract(r.text) or r.text[:3000]
+            if len(content) > MAX_CONTENT_SIZE:
+                return content[:MAX_CONTENT_SIZE]
+            return content
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)